bentoml/_internal/cli/containerize.py

```diff
@@ -246,6 +250,21 @@ def containerize(  # type: ignore
         else:
             output_ = None
 
+        if not platform:
+            load = True
+        else:
+            if len(platform) > 1:
+                logger.warning(
+                    "Multiple `--platform` is parsed. "
+                    "Make sure to also use `--push` to push images "
+                    "to a repository, otherwise, image will only be saved "
+                    "under Docker build cache. "
+                    "Refers to https://docs.docker.com/engine/reference/commandline/buildx_build/#load "
+                    "for more information."
```

Comment:
Should just check whether `--push` was passed and not print this if it was?

```suggestion
                    "Multiple `--platform` arguments were found.  "
                    "Make sure to also use `--push` to push images "
                    "to a repository or generated images will not be saved. "
                    "For more information, see https://docs.docker.com/engine/reference/commandline/buildx_build/#load."
```